backend/core/src/seeds/ami-charts/default-ami-chart-san-mateo.ts

```diff
@@ -0,0 +1,296 @@
+import { AmiChartDefaultSeed } from "./default-ami-chart"
+import { CountyCode } from "../../shared/types/county-code"
+
+export class AmiDefaultSanMateo extends AmiChartDefaultSeed {
```

Comment:
The data in this chart looks identical apart from the name to the chart above, is there a way to share the data?